EndlessSky.cbp

```diff
@@ -77,6 +77,82 @@
 			<Add library="C:\Program Files\mingw64\x86_64-w64-mingw32\lib\libopengl32.a" />
 			<Add directory="C:/dev64/lib" />
 		</Linker>
+		<Unit filename="data/boarding missions.txt" />
```

Comment:
Compiling the game does not depend on these text files at all, so they should not be present